common/Contracts/IDevHomeActionRender.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AdaptiveCards.ObjectModel.WinUI3;
+using AdaptiveCards.Rendering.WinUI3;
+using DevHome.Common.Renderers;
+using ActionMode = AdaptiveCards.ObjectModel.WinUI3.ActionMode;
+
+namespace DevHome.Common.Contracts;
+
+public interface IDevHomeActionRender : IAdaptiveElementRenderer
```

Comment:
ACs have IAdaptiveElementRenderer and IAdaptiveActionRenderer. This seems to be an interface for elements, so it would be called IDevHomeElementRenderer.

Reply:
Actually this one is suppose to just be the custom renderer for an ActionSet but I can update the name